workspaces/libnpmpublish/test/publish.js

```diff
@@ -9,6 +9,7 @@ const t = require('tap')
 
 const MockRegistry = require('@npmcli/mock-registry')
 const mockGlobals = require('../../../test/fixtures/mock-globals.js')
+const { generateProvenance } = require('../lib/provenance.js')
```

Comment:
We are avoiding unit tests here. If the code can't be reached from `require('libnpmpublish')` it's code that shouldn't exist.

Reply:
Done.